regtest.sh

```diff
@@ -261,8 +307,12 @@ run-seq() {
 
   write-test-cases
 
+  # Generate parameters for all test cases.
+  cat $spec_list \
```

Comment:
distinguish case list and instance list:

andychu@andychu-2:~/git/rappor$ lines a b c | python -c '

python -c '
import sys
 for line in sys.stdin:
   for i in xrange(1, 6):
     print line.strip(), i
'